charts/project-origin-wallet/values.yaml

```diff
@@ -99,12 +99,15 @@ registries: []
 # persistence defines the persistence configuration for the wallet server
 persistance:
 
+  # enabled determines if the cloud native postgresql instance is enabled
+  enabled: true
```

Comment:
Remove this. 

Earlier we used an enabled flag in each section, but as you also spotted, this is not great since multiple could be enabled, which does not make sense. And disabling persistance is not an option.

Reply:
I was actually debating whether it should be included or not. I don't know if Wallet is able to operate without a database, and instead defaults to using some in-memory implementation?

Reply:
Done. I have also in this commit added some additional details about database setup:

https://github.com/project-origin/wallet/pull/254/commits/e9dde96d1059cad22e4028a3039744b0822df2bd